src/components/markdown-renderer/markdown-renderer.tsx

```diff
@@ -244,12 +243,12 @@ export const MarkdownRenderer: React.FC<MarkdownRendererProps> = ({ content, onM
       permalinkSymbol: '<i class="fa fa-link"></i>'
     })
     md.use(mathJax({
```

Comment:
do we want to rename this component, too?

Reply:
We can rename the import, but not the lib.